src/modelgauge/suts/together_client.py

```diff
@@ -29,7 +32,7 @@ def _retrying_post(url, headers, json_payload):
     """HTTP Post with retry behavior."""
     session = requests.Session()
     retries = Retry(
-        total=6,
+        total=10,
```

Comment:
Dang, did you notice a difference after increasing this number?

Reply:
I did. This was based on me doing runs, getting failures, and tuning until we stopped getting failures.